test/functional/wallet_bumpfee.py

```diff
@@ -286,6 +287,39 @@ def test_bumpfee_with_descendant_fails(self, rbf_node, rbf_node_address, dest_ad
     self.clear_mempool()
 
 
+def test_bumpfee_with_abandoned_descendant_succeeds(self, rbf_node, dest_address):
+    self.log.info('Test that fee can be bumped when it has abandoned descendant')
+    parent_id = spend_one_input(rbf_node, dest_address, Decimal("0.00049000"))
+    # Find change output
+    output_list = rbf_node.getrawtransaction(parent_id, 1)["vout"]
+    change_output = next(o for o in output_list if o["value"] == Decimal("0.00049000"))
+
+    # Submit child transaction with low fee
+    child = rbf_node.createrawtransaction([{"txid": parent_id, "vout": change_output["n"]}], {dest_address: 0.0004850})
```

Comment:
Might be easier to use the `send` RPC?

Reply:
Happy to change it, but I picked this because `send` is marked as "EXPERIMENTAL warning: this call may be changed in future releases". I do see other uses of it in tests, though.

Reply:
I think it's fine to use it in tests. If we ever change the interface, we'll have to refactor the test.

Reply:
Great. I updated the test to use `send` and it did end up simplifying the test code a bit. Thanks!